keyboards/freyr/info.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
Could you add the TKL ansi and is layouts here, as well?

Reply:
Oh, sorry about this.
i will add right now